packages/core/src/types/IViewport.ts

```diff
@@ -105,6 +106,61 @@ export type ViewReference = {
   bounds?: BoundsLPS;
 };
 
+/**
+ * A view presentation stores information about how the view is presented to the
+ * user, such as rotation, the displayed area, pan/zoom etc.
+ * When used for choosing a presentation to return, set the units to the desired
+ * unit type in order to include that value.  Of course some viewports will not
+ * return some unit values at all, such as Stack and slabThickness.
+ */
+export type ViewPresentation = {
+  /**
+   * The slice thickness - in frames(true/default) it will be 1 for a frame distance of
+   * 1 pixel thickness, while for mm will be in mm distance.
+   */
+  slabThickness?: number;
+
+  /**
+   * The rotation of the view - this is related to cameraViewUp, but is relative
+   * to the viewNormal and the default viewUp for that viewNormal.
+   */
+  rotation?: number;
+
+  /**
+   * The display area being shown.  This is more consistent than applying a set
+   * of boundary areas.
+   */
+  displayArea?: DisplayArea;
+
+  /**
+   * The zoom value is a zoom factor relative either to fit to canvas or relative
+   * to the display area.
+   * The default true units are relative to the initial camera
+   * scale to fit is used to get units relative to the scale to fit camera.
+   */
+  zoom?: number;
+
+  /**
+   * The pan value is how far the pan has moved relative to the fit to canvas
+   * or relative to the display area initial position/sizing.
+   * true is the default units, which is relative to the initial canvas setting,
+   * in zoom relative units.
+   */
+  pan?: Point2;
+};
+
+export type ViewPresentationSelector = {
+  slabThickness?: boolean;
+  // Camera relative parameters
+  rotation?: boolean;
+  displayArea?: boolean;
+  zoom?: boolean;
+  pan?: boolean;
+  // Transfer function relative parameters
+  windowLevel?: boolean;
+  paletteLut?: boolean;
+};
```

Comment:
can you please document this, what is a viewPresentation selector and how is it different from viewPresentation, then each item requires doc please